src/main/java/com/conveyal/datatools/manager/jobs/FeedExpirationNotificationJob.java

```diff
@@ -13,20 +14,21 @@
 import static com.conveyal.datatools.manager.auth.Auth0Users.getVerifiedEmailsBySubscription;
 import static com.conveyal.datatools.manager.utils.NotificationsUtils.sendNotification;
 
-public class FeedExpirationNotificationJob implements Runnable {
+public class FeedExpirationNotificationJob extends MonitorableJob {
```

Comment:
My thinking was that it would be good to limit the objects stored in `ScheduledJob#job` to only `MonitorableJob` subclasses, but I see that these expiration notifications could stack up to a large number and may not be something we want to consider with the other monitorable jobs. Changing this would require making the `ScheduledJob#job` type a `Runnable`. Do you see any issue there?

Reply:
The issue is mainly the overhead of using a custom class instead of the built-in Runnable class.  I don't have strong feelings about this, so I guess this is ok.